src/rum.js

```diff
@@ -49,12 +49,13 @@ if (!rumConfig.serviceName) {
 if (!rumConfig.serverUrl) {
     rumConfig.serverUrl = serverUrl
 }
-if (!rumConfig.serviceVersion) {
-    rumConfig.serviceVersion = serviceVersion
-}
-rumConfig.logLevel = 'debug'
 
-var apm = initApm(rumConfig)
+const apm = initApm(Object.assign({}, rumConfig, {
+    serviceVersion,
```

Comment:
The priority of the `serviceVersion` is changed.  before `rumConfig.serviceVersion` would be used if both that and `serviceVersion` had values, but here it would be overwritten.

Reply:
Yeah we don't even set anything anywhere so not a huge concern, but anyway I will revert it.